infertrade/utilities/operations.py

```diff
@@ -136,10 +204,15 @@ def research_over_price_minus_one(x: Union[np.ndarray, pd.Series], shift: int) -
 
 class PricePredictionFromSignalRegression(TransformerMixin, BaseEstimator):
 
-    """Class for creating price predictions from signal values."""
+    """This class creates price predictions from signal values.
+
+    Args:
+        market_to_trade: The name of the column which contains the historical prices.
+
+    """
 
     def __init__(self, market_to_trade: str = None):
-        """We create by determining one input column as being the price to target."""
+        # We create by determining one input column as being the price to target.
```

Comment:
Could keep as docstring for the construction method?

Reply:
Do you mean just for this particular __init__ method or also for the two whose docstring just says "Trivial creation method"?

Reply:
`The class constructor should be documented in the docstring for its __init__ method. Individual methods should be documented by their own docstring.`

Is the recommendation of [Pep257](https://www.python.org/dev/peps/pep-0257/)

Reply:
So typically we can just say "Construction method for class X" in the docstring? Or similar?

Reply:
:+1: I was going off a bad example then. I'll fix it.